yt_dlp/plugins.py

```diff
@@ -24,6 +24,7 @@
 
 PACKAGE_NAME = 'yt_dlp_plugins'
 COMPAT_PACKAGE_NAME = 'ytdlp_plugins'
+_EXTRACTOR_PLUGIN_OVERRIDES = []
```

Comment:
keeping this private for time-being as likely would change in the future.

any ideas for better location or solution?